### PR TITLE
Freon no longer dissapates

### PR DESCRIPTION
**Pull request by Cyberboss**
Make meme gas actually useful for cooling the SM

:cl: Cyberboss
tweak: Freon will no longer evaporate
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
But it freezes over the vents so it doesnt' actually get plasma or anything else out, making it useless for cooling.

**Comment by YakumoChen** [comment]
Also makes releasing the meme gas anywhere else more cancerous then ever, unmeltable ice floors are awful.

**Comment by CPTANT** [comment]
ohw yay, we STILL have permanent ice.

Seriously, that ice is more snowflake than a snowman.

Freons ice code should be removed, things should freeze when they get cold and thaw when they get hot. Freon should just reduce the temperature of the room it is in.

**Comment by XDTM** [comment]
ice does thaw when it gets hot now, or at least i saw a PR that did that get merged.

**Comment by Cyberboss** [comment]
1. Ice isn't permanent, it's melted by heat.

2. Freezing stuff is a purely visual effect

**Comment by Cyberboss** [comment]
Except with icy floors

**Comment by CPTANT** [comment]
@XDTM 

Apparently you are right, PR #22054 of that kind was merged, however it states that things not on an open turf stay frozen anyway

**Comment by Cyberboss** [comment]
That's actually not the problem, as I've found out. I still don't know what
keeps object from occasionally unfreezing.

File: code/modules/atmospherics/gasmixtures/gas_mixture.dm [hunk 1]
```diff
@@ -183,7 +183,7 @@ var/list/gaslist_cache = null
 		if(cached_gases["freon"])
 			if(cached_gases["freon"][MOLES] >= MOLES_PLASMA_VISIBLE)
 				if(holder.freon_gas_act())
-					cached_gases["freon"][MOLES] -= MOLES_PLASMA_VISIBLE
+					cached_gases["freon"][MOLES] -= 0.1
```
Comment (Iamgoofball) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: Remove it entirely please I keep forgetting to now that scrubbers work on it properly

**Review by Iamgoofball — APPROVED** [review]

**Comment by Iamgoofball** [comment]
yakumo is literally spouting bullshit ignore their autistic anime obsessed posts

**Comment by coiax** [comment]
I am in favour of removing freon, and just freezing things when they get cold. A magical "COLD GAS" seems to sidestep the whole point of gas temperatures.

**Comment by Tokiko1** [comment]
Not a fan of this because this causes an infinite atmospheric chain reaction where freon will keep cooling surrounding gas, causing it to drop in pressure which in turn will cause the freon to spread further. 

Also, the can of freon near the engine is meant for emergencies, like when the engine reaches 80% instability. It is not meant to be a magical infinitely self cooling gas that completely removes the need for any cooling heat exchangers or freezers. Or even any gas circulation actually.

**Comment by Iamgoofball** [comment]
@coiax it'd be too overpowered it's why it's locked to freon, which balances it, realisms can fuck themselves

@Tokiko1 what is scrubbers

**Comment by Robustitron** [comment]
@Iamgoofball

Scrubbers won't save you when 3-4 dudes run into engineering and hug the supermatter at roundstart.

That happened last saturday, I tried to stabilize the engine while everyone else in the room panicked, siphoning and scrubbing didn't do shit at that point.

We got to 90% instability when (I think) oldman connected the freon, at 95% we had to shut down the scrubbers because even the freon wouldn't work unless we just trapped it in there.

Anyways if freon goes we need a replacement or an eject like all those other stations.

**Comment by Iamgoofball** [comment]
fucking hell

seems like the only way to make the engine go nuclear is to suicide with it

wtf

**Comment by optimumtact** [comment]
or you know, just be competent and not suck

**Comment by Tokiko1** [comment]
I've changed my opinion on this PR. If people feel that it is easy enough to scrub freon out then why not?

**Comment by optimumtact** [comment]
It should decay slower but should still eventually decay.

This isn't about the engine, it's about how much of a pain it is to scrub freon out during spills and malicious openings

**Comment by Iamgoofball** [comment]
@optimumtact It scrubs the same as any other gas.

**Pull request closed without merging**